src/smc-webapp/projects.cjsx

```diff
@@ -1164,7 +1205,7 @@ exports.ProjectTitle = ProjectTitle = rclass
 
     render : ->
         if not @props.project_map?
-            return <Loading />
```

Comment:
Why did you make this change?  It got to be wrong.  It means that now if the project_map isn't loaded yet, instead of showing "Loading" it will never show loading and will instead show "(private project)", even if it isn't private.  I would just change it, but I'm puzzled since I don't know why you made the change.